Jenkinsfile

```diff
@@ -60,12 +60,37 @@ pipeline {
                 not { branch 'develop' }
                 not { branch 'master' }
             }
-            steps { 
+            steps {
                 script {
                     utils.killOldBuilds()
                 }
             }
         }
+        stage("Make-doxygen") {
+            agent any
+            when { branch 'PR-795'}
```

Comment:
Want to change this to 'master'?

Reply:
Sure, I'll also move it to the bottom of Jenkinsfile